CHANGELOG

```diff
@@ -1,12 +1,17 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
-## [4.0.0] - 2024-03-06
+## [4.0.0] - 2024-08-09
 # Removed
 - Support of tasks as it unused to due its incompatibility with GitHub.
 
 # Added
 - Bert-E's status notifications through a build status check.
+- Support of development branches with only major version: `development/x`.
+
+# Changed
+- Integration queue branches pattern is now `q/w/{pr.id}/{integration.branch}`
```

Comment:
Let's make sure we communicate about that; I've looked around through https://github.com/search?q=org%3Ascality+q%2F+language%3Ayaml+path%3A%2F%5E%5C.github%5C%2F%2F&type=code and indeed I don't think it's going to be an issue.